src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -460,6 +460,7 @@ public override bool CanApplyChange(ApplyChangesKind feature)
                 case ApplyChangesKind.ChangeAnalyzerConfigDocument:
                 case ApplyChangesKind.AddSolutionAnalyzerReference:
                 case ApplyChangesKind.RemoveSolutionAnalyzerReference:
+                case ApplyChangesKind.ChangeProjectInfo:
                     return true;
```

Comment:
We shouldn't be saying this unless we actually implement the support for it. If we don't want to do that in this PR (it might go down into a huge path of weeds and sadness) that's fine, but we should keep this returning false.